src/website/app/pages/RenderProps/render-props.md

```diff
@@ -0,0 +1,159 @@
+## Overview
+
+_Excerpted from the official [React docs](https://reactjs.org/docs/render-props.html)._
+
+> The term "render prop" refers to a simple technique for sharing code between
+> React components using a prop whose value is a function.
+>
+> A component with a render prop takes a function that returns a React element
+> and calls it instead of implementing its own render logic.
+>
+> ```jsx
+> <DataProvider render={data => (
+>   <h1>Hello {data.target}</h1>
+> )} />
+> ```
+>
+> _It’s important to note that just because the pattern is called
+> "render props" it does not mean that the prop will always be named render.
+> In fact, any prop that is a function that a component uses to know what to
+> render is technically a "render prop"._
```

Comment:
I think this last paragraph would be better suited outside the blockquote since it's not a direct quote.

Reply:
Sure, I had it that way originally, and can change it back.